charts/fediscoverer/templates/deployment.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
We need a second deployment for the workers

Reply:
You can find the deployment instructions in this non-yet-merged PR: https://github.com/mastodon/fediscoverer/pull/25

Reply:
It is not sidekiq but https://github.com/rails/solid_queue. SolidQueue keeps the job data in the database which is why we do not need redis here (yet).